frontend/src/metabase/parameters/components/widgets/TemporalUnitWidget/TemporalUnitWidget.tsx

```diff
@@ -19,22 +17,21 @@ export function TemporalUnitWidget({
 }: TemporalUnitWidgetProps) {
   const availableUnits =
     parameter.temporal_units ?? Lib.availableTemporalUnits();
+  const availableItems = availableUnits.map(unit => ({
```

Comment:
memo?

Reply:
Not needed - this is a short lived component in a popover